devops/charts/server/values_prod.yaml

```diff
@@ -1,12 +1,12 @@
 # Default values for vdr-proxy prod deployment
 namespace: c2a2c4-prod
 
-replicaCount: 3
+replicaCount: 2
```

Comment:
For prod the convention is usually 3 minimum. Maybe this is outdated and 2 is good enough so if you've read elsewhere that 2 is fine the LGTM.